tests/wot_coap_generation/config/wot_td/thing_model.json

```diff
@@ -6,13 +6,25 @@
     "properties": {
         "status": {
             "description": "current status of the lamp (on|off)",
+            "descriptions": {
+                "de": "Aktueller Lampen status"
+            },
             "type": "string",
             "readOnly": true
         }
     },
     "actions": {
         "toggle": {
-            "description": "Turn the lamp on or off"
+            "description": "Turn the lamp on or off",
+            "descriptions": {
+                "de": "Schalte die Lampe aus und an"
+            }
+        },
+        "rmtoggle": {
+            "description": "Toggle endpoint",
+            "descriptions": {
+                "de": "Toggle Endpoint hinzufuegen"
+            }
```

Comment:
So this endpoint enables you to toggle the toggle? #toggleception ;)

Reply:
Yes ^^ I know that's a little confusing. I guess you get the point of having that. Maybe you can come up with some simple different example. I thought more of having configurations which enable/disable different functionality.